src/AppInstallerRepositoryCore/Microsoft/Schema/Portable_1_0/PortableTable.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "PortableTable.h"
+#include "SQLiteStatementBuilder.h"
+
+namespace AppInstaller::Repository::Microsoft::Schema::Portable_V1_0
+{
+    using namespace std::string_view_literals;
+    static constexpr std::string_view s_PortableTable_Table_Name = "portable"sv;
+    static constexpr std::string_view s_PortableTable_Index_Separator = "_"sv;
+    static constexpr std::string_view s_PortableTable_Index_Suffix = "_index"sv;
+
+    static constexpr std::string_view s_PortableTable_FilePath_Column = "filePath"sv;
+    static constexpr std::string_view s_PortableTable_FileType_Column = "fileType"sv;
+    static constexpr std::string_view s_PortableTable_SHA256_Column = "sha256"sv;
+    static constexpr std::string_view s_PortableTable_SymlinkTarget_Column = "symlinkTarget"sv;
+    static constexpr std::string_view s_PortableTable_IsCreated_Column = "isCreated"sv;
+
+    std::string_view PortableTable::TableName()
+    {
+        return s_PortableTable_Table_Name;
+    }
+    
+    void PortableTable::Create(SQLite::Connection& connection)
+    {
+        using namespace SQLite::Builder;
+
+        SQLite::Savepoint savepoint = SQLite::Savepoint::Create(connection, "createPortableTable_v1_0");
+
+        StatementBuilder createTableBuilder;
+        createTableBuilder.CreateTable(s_PortableTable_Table_Name).BeginColumns();
+
+        createTableBuilder.Column(ColumnBuilder(s_PortableTable_FilePath_Column, Type::Text).NotNull());
+        createTableBuilder.Column(ColumnBuilder(s_PortableTable_FileType_Column, Type::Int64).NotNull());
+        createTableBuilder.Column(ColumnBuilder(s_PortableTable_SHA256_Column, Type::Blob).NotNull());
+        createTableBuilder.Column(ColumnBuilder(s_PortableTable_SymlinkTarget_Column, Type::Text));
+        createTableBuilder.Column(ColumnBuilder(s_PortableTable_IsCreated_Column, Type::Bool).NotNull());

```

Comment:
Could there be comments with how these are planned to be used?  I'm not sure how `IsCreated` is to be used, or why it couldn't be a file type flag.